src/hetrf.cc

```diff
@@ -34,6 +34,12 @@ void hetrf(
     using BcastList  = typename Matrix<scalar_t>::BcastList;
     using ReduceList = typename Matrix<scalar_t>::ReduceList;
 
+    // Constants
+    const scalar_t zero = 0.0;
+    const scalar_t one  = 1.0;
+    const int64_t ione  = 1;
+    const int64_t izero = 0;
+    const int priority_1 = 1;
```

Comment:
Is there a reason for having only priority_1, meaning no other priority is defined/used.

Reply:
Other tasks run at the default priority, which is 0 (lowest priority).